CollapseLauncher/Classes/Extension/TaskExtensions.cs

```diff
@@ -50,7 +131,7 @@ internal static async Task<TResult?>
                     consolidatedToken = CancellationTokenSource.CreateLinkedTokenSource(innerCancellationToken.Token, fromToken);
 
                     ActionTimeoutTaskCallback<TResult?> delegateCallback = funcCallback();
-                    return await delegateCallback(consolidatedToken.Token);
+                    return await delegateCallback(consolidatedToken.Token).ConfigureAwait(false);
```

Comment:
Make sure to test the case with this, since for the past, uhh, ConfigureAwait seems to always be a problem that detaches the UI thread for some reason. If no need, can remove or use other UI-aware awaiter.